blueocean-dashboard/src/main/less/core.less

```diff
@@ -64,7 +64,7 @@
     margin-top: 10px;
 
     th {
-        width: 75px;
+        width: 100px;
```

Comment:
This will do for now. When we switch the table layouts over (hopefully this week) we'll need to take the number of extensions into account when sizing action-button-y columns. Luckily, the (baseline) column widths are runtime, not CSS in the new flexbox-based system.